src/js/form-builder.js

```diff
@@ -936,6 +936,7 @@ function FormBuilder(opts, element, $) {
 
     let attrVal = values[attribute] || ''
     let attrLabel = mi18n.get(attribute) || attribute
+    const typeVal = values['type'] || ''
```

Comment:
```suggestion
```

I don't think you need to define an extra variable here. The type can never be undefined nor and we only reference it once in the code below.

Reply:
Yup it left over from the previous code amendments.  No longer required.